package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "app-overview",
+  "version": "0.0.3",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --config webpack.config.js",
+    "start": "webpack-dev-server --progress --colors",
+    "rand": "./random.js",
+    "precommit": "lint-staged"
+  },
+  "lint-staged": {
+    "*.{js,json,css,md}": ["prettier --write", "git add"]
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/rainierlouis/overview.git"
+  },
+  "keywords": ["overview"],
+  "author": "",
+  "bin": {
+    "overview": "./index.js"
+  },
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/rainierlouis/overview/issues"
+  },
+  "homepage": "https://github.com/rainierlouis/overview#readme",
+  "devDependencies": {
```

Comment:
not entirely sure if all of these are devDependencies or just some of them. Have you checked if you can install the node package after the webpack build with no issues? I'm asking since you use some of these dependencies in the final product, so node should require them to be installed alongside your package.